src/test/java/fi/metatavu/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -109,32 +318,50 @@ private void printServiceChannelList(ServiceChannelType type, File[] files) thro
     System.out.println("    };");
   }
 
-  @SuppressWarnings("unused")
   private PtvDownload downloadOrganizations() throws IOException {
-    File[] organizationFiles = listJsonFiles(getOrganizationsFolder());
+    File organizationsFolder = getOrganizationsFolder();
+    
+    File[] organizationFiles = listJsonFiles(organizationsFolder);
     for (File organizationFile : organizationFiles) {
+      String organizationId = FilenameUtils.getBaseName(organizationFile.getName());
+      downloadOrganization(organizationsFolder, organizationId);
+      limitOrganizationServices(organizationFile);
       downloadOrganizationServices(organizationFile);
     }
     
     return this;
   }
   
+  private void downloadOrganization(File organizationsFolder, String organizationId) {
+    String url = getFullUrl(String.format("/Organization/%s", organizationId));
+    File target = new File(organizationsFolder, String.format("%s.json", organizationId));
+    if (target.exists()) {
+      target.delete();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use "java.nio.Files#delete" here for better messages on error conditions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4042)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "delete". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS899)